python/generate/templates/api.mustache

```diff
@@ -107,10 +110,10 @@ class {{classname}}(object):
                               request; this effectively ignores the authentication
                               in the spec for a single request.
         :type _request_auth: dict, optional
-        :return: Returns the result object.
+        :return: Returns the result object, the HTTP status code, and the headers.
                  If the method is called asynchronously,
                  returns the request thread.
-        :rtype: {{#returnType}}tuple({{returnType}}, status_code(int), headers(HTTPHeaderDict)){{/returnType}}{{^returnType}}None{{/returnType}}
+        :rtype: {{#returnType}}({{returnType}}, int, HTTPHeaderDict){{/returnType}}{{^returnType}}None{{/returnType}}
```

Comment:
Looks great, not sure what the `status_code` and `headers` in the `rtype` where supposed to represent? This seems like a much more suitable approach and as you say is much more enjoyable to work with in PyCharm and likely other IDEs as well.